UI/Web/src/app/admin/manage-email-settings/manage-email-settings.component.html

```diff
@@ -1,16 +1,16 @@
 <div class="container-fluid">
     <form [formGroup]="settingsForm" *ngIf="serverSettings !== undefined">
         <h4>Email Services (SMTP)</h4>
-        <p>Kavita comes out of the box with an email service to power flows like invite user, forgot password, etc. Emails sent via our service are deleted immediately. You can use your own
-            email service, by setting up <a href="https://github.com/Kareadita/KavitaEmail" target="_blank" rel="noopener noreferrer">Kavita Email</a> service. Set the url of the email service and use the Test button to ensure it works. 
-            At any time you can reset to the default. There is no way to disable emails for authentication, although you are not required to use a 
-            valid email address for users. Confirmation links will always be saved to logs and presented in the UI. 
-            Registration/Confirmation emails will not be sent if you are not accessing Kavita via a publicly reachable url.
-            <span class="text-warning">If you want Send To device to work, you must host your own email service.</span>
+        <p>Kavita comes out of the box with an email service to power tasks like inviting users, password reset requests, etc. Emails sent via our service are deleted immediately. You can use your own
+            email service by setting up the <a href="https://github.com/Kareadita/KavitaEmail" target="_blank" rel="noopener noreferrer">Kavita Email</a> service. Set the URL of the email service and use the Test button to ensure it works.
+            You can reset these settings to default at any time. There is no way to disable emails for authentication, although you are not required to use a
+            valid email address for users. Confirmation links will always be saved to logs and presented in the UI.
+            Registration/confirmation emails will not be sent if you are not accessing Kavita via a publicly reachable URL.
```

Comment:
Can you reword this in retrospect to include that:
Registration/confirmation emails will not be sent if you are not accessing Kavita via a publicly reachable URL or unless Host Name is configured.

Host name is a new feature that allows you to specify the reverse proxy used to access Kavita and hence Kavita will assume that for email generation and override the check if accessible.